python/apps/taiga/src/taiga/workflows/services/__init__.py

```diff
@@ -112,3 +109,20 @@ async def get_status(workflow_id: UUID, status_slug: str) -> WorkflowStatus | No
         },
         select_related=["workflow"],
     )
+
+
+##########################################################
+# update workflow status
+##########################################################
+
+
+async def update_workflow_status(status: WorkflowStatus, values: dict[str, Any] = {}) -> WorkflowStatus:
+    # TODO: Emit events
+
+    if not values:
+        return status
+
+    if "name" in values and values["name"] is None:
```

Comment:
this validation doesn't need to hit the database, cannot it be in the validator itself?